src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -68,7 +76,25 @@ const DomainList = (props) => {
           </View>
         </View>
       </TouchableOpacity>
-      {item.isunfollowed ? (
+      {isBlockedSection ? (
+        item.isUnblocked ? (
+          <TouchableNativeFeedback testID="isUnblock" onPress={handleSetBlock}>
+            <View style={styles.followContainer}>
+              <View style={styles.buttonBlockUser}>
+                <Text style={styles.textButtonBlockUser}>Block</Text>
+              </View>
+            </View>
+          </TouchableNativeFeedback>
+        ) : (
+          <TouchableNativeFeedback testID="isBlock" onPress={handleSetUnblock}>
+            <View style={styles.followContainer}>
+              <View style={styles.buttonBlock}>
+                <Text style={styles.textButtonBlock}>Blocked</Text>
+              </View>
+            </View>
+          </TouchableNativeFeedback>
+        )
+      ) : item.isunfollowed ? (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems like the `TouchableNativeFeedback` component is being used for both iOS and Android platforms. However, this component is not fully compatible with iOS and may cause issues. Consider using a platform-specific approach or a cross-platform touchable component like `TouchableOpacity`.
```diff
          <TouchableNativeFeedback testID="isUnblock" onPress={handleSetBlock}>
- 82:             <View style={styles.followContainer}>
+ 82:             <TouchableOpacity style={styles.followContainer} testID="isUnblock" onPress={handleSetBlock}>
              <View style={styles.buttonBlockUser}>
                <Text style={styles.textButtonBlockUser}>Block</Text>
              </View>
            </View>
- 87:           </TouchableNativeFeedback>
+ 87:           </TouchableOpacity>

          <TouchableNativeFeedback testID="isBlock" onPress={handleSetUnblock}>
- 90:             <View style={styles.followContainer}>
+ 90:             <TouchableOpacity style={styles.followContainer} testID="isBlock" onPress={handleSetUnblock}>
              <View style={styles.buttonBlock}>
                <Text style={styles.textButtonBlock}>Blocked</Text>
              </View>
            </View>
- 95:           </TouchableNativeFeedback>
+ 95:           </TouchableOpacity>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->